### DISCUSSION TITLE
[Feature] <Support use leaflet-providers to switch the base map >

### DISCUSSION THREAD

**Discussion by wuqi** [immich-app/immich#3755] [Feature Request]
### The feature

Can you please support  use  the [leaflet-providers](https://github.com/leaflet-extras/leaflet-providers) to switch the base map？Like https://leaflet-extras.github.io/leaflet-providers/preview/ 

#2426   mentioned  dark mode for map but did not mention a solution,I think leaflet-providers could solve this problem.,it support you change the base map yourself.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
You can already choose different light/dark mode.

![image](https://github.com/immich-app/immich/assets/27055614/3ed82cd2-df5d-4a72-bf77-e07b2b1b7cf5)

**Reply by bo0tzz** [reply]
leaflet-providers also allows switching between many other types of maps (eg satellite images, etc). Do we want to keep this FR open to eventually add support for that?

**Comment by bo0tzz** [comment]
#4294 switches to Maplibre and allows configuring the base URL, so it will close this FR.

**Comment by bo0tzz** [comment]
Closed by #4294
